app/assets/javascripts/preferences/panes/two-factor-auth/TwoFactorAuthView.tsx

```diff
@@ -28,26 +27,25 @@ export const TwoFactorAuthView: FunctionComponent<{
           <Text>
             An extra layer of security when logging in to your account.
           </Text>
+          {auth.errorMessage != null && (
+            <Text className="color-dark-red">{auth.errorMessage}</Text>
```

Comment:
We should be using StyleKit colors/classes. dark-red is subjective and relative to the theme. Should be SK `danger` class.

Reply:
That's a different nuance of red.

Reply:
Nothing should be colored without StyleKit in general. Otherwise users can't customize appearance.

Reply:
The 'color-dark-red' is defined in StyleKit. I understand what you mean, that the name should match the context. But seeing as danger was already taken, I couldn't find a different appropriate name that wouldn't overshadow danger's meaning. Shall I use "color-danger" instead, and we update the color schemes later?

Reply:
Used color-danger :)